src/unstract/sdk/adapters/x2text/llm_whisperer/src/llm_whisperer.py

```diff
@@ -360,22 +371,36 @@ def _write_output_to_file(self, output_json: dict, output_file_path: Path) -> No
         try:
             text_output = output_json.get("text", "")
             logger.info(f"Writing output to {output_file_path}")
-            output_file_path.write_text(text_output, encoding="utf-8")
+            fs.write(
+                path=output_file_path.name,
+                mode="w",
+                encoding="utf-8",
+                data=text_output,
+            )
+            # output_file_path.write_text(text_output, encoding="utf-8")
             try:
                 # Define the directory of the output file and metadata paths
                 output_dir = output_file_path.parent
                 metadata_dir = output_dir / "metadata"
                 metadata_file_name = output_file_path.with_suffix(".json").name
                 metadata_file_path = metadata_dir / metadata_file_name
                 # Ensure the metadata directory exists
-                metadata_dir.mkdir(parents=True, exist_ok=True)
+                fs.mkdir(metadata_dir, create_parents=True)
+                # metadata_dir.mkdir(parents=True, exist_ok=True)
```

Comment:
remove this commented lines if not required

Reply:
Done